state/peerAccountsDB.go

```diff
@@ -62,10 +63,18 @@ func (adb *PeerAccountsDB) SnapshotState(rootHash []byte) {
 	log.Trace("peerAccountsDB.SnapshotState", "root hash", rootHash)
 	trieStorageManager := adb.mainTrie.GetStorageManager()
 
+	stats := &snapshotStatistics{
```

Comment:
the constructor can have an int parameter for the initial Add method call for the wait group. Here you would need to call with the 0 value

Reply:
Done.